### DISCUSSION TITLE
how to run global find-replace?

### DISCUSSION THREAD

**Discussion by aborruso** [nushell/nushell#3475] [Q&A] [answered]
Hi,
if I run

```
echo "ciao bau" | str find-replace "a" "000"
```

it replaces the first `a` and I have `ci000o bau`. How to obtain `ci000o b000u`?

**Comment by sophiajt** [comment, marked as answer]
You can find more info about a command by calling `--help` on it, like `str find-replace --help`.

In the help, you'll be able to get examples for how to use it, too.

**Reply by aborruso** [reply]
Thank you again @jonathandturner 

For me this is an example for the subcommands webpage I was looking for. I will use the shell!
